### PR TITLE
Minor fix for the raw butter recipe

### PR DESCRIPTION
**Pull request by Playyne**
#### Summary
None

#### Purpose of change

A bug was introduced in #51527 that made the shake recipe for raw butter uncraftable. The tool entry expects charges from the glass jar, which doesn't have any.

#### Describe the solution

Setting the charge entry to -1 makes the recipe ignore charges when considering tools, so the glass jar works as a tool again.

#### Describe alternatives you've considered
Using more keystrokes than this edit took to craft a butter churn and use that instead.

#### Testing

I made the edit on my local install and launched the game, recipe is now properly craftable.

### CODE REVIEW

File: data/json/recipes/food/dairy_products.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "time": "20 m",
     "book_learn": [ [ "dairy_book", 3 ] ],
     "//": "Book Things to do with milk.  Add curdled milk and cheese recipes to the book.  Also consider adding to brewing json Airag from this book.",
-    "tools": [ [ [ "jar_glass_sealed", 1 ] ] ],
+    "tools": [ [ [ "jar_glass_sealed", -1 ] ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/dairy_products.json:32]: This recipe should have a byproduct of four buttermilk as well.

Comment (Playyne) [reply data/json/recipes/food/dairy_products.json:32]: The original recipe took this into account already. Line 23.

Comment (Maleclypse) [reply data/json/recipes/food/dairy_products.json:32]: I must have been blind when I read the recipe looking for that.   :)

Comment (Playyne) [reply data/json/recipes/food/dairy_products.json:32]: No problem.

**Pull request merged**